resolwe_bio/docker_images/dnaseq/packages-manual/r-packages.sh

```diff
@@ -2,18 +2,21 @@
 
 . /var/cache/build/packages-manual/common.sh
 
-# These steps need to be performed because certain packages produce a "database corrupt error". The solution is to
-# restart R, but this is not feasible during the docker built or building of dependencies. This chunk also installs
-# packages from bioconductor. This entire step should be considered temporary until we can get packrat to work for
-# packages from all universes (CRAN, Bioconductor).
-Rscript --slave --no-save --no-restore-history -e "install.packages('assertthat')"
+# Pin BiocManager version 3.13
+Rscript --slave --no-save --no-restore-history -e "install.packages('BiocManager')"
```

Comment:
`BiocManager` is installed by `renv` when you refer to a package using `bioc::<package>` on installation. >ou can omit this since we don't care what version of `BiocManager` gets installed.